Services/Interfaces/web_interface/templates/profiles_selector.html

```diff
@@ -34,43 +35,56 @@ <h3>
         </div>
     </div>
     <div class="card-body pt-0">
-        {% if current_logged_in_email %}
-        <div class="alert alert-dark row py-0 my-1">
-            <div class="col-8 my-auto">
-                Logged in as {{current_logged_in_email}}
-                {% if selected_user_bot["name"] %}
-                    using bot <span class="badge badge-info">{{ selected_user_bot["name"] }}</span>
-                {% else %}
-                    <span class="badge badge-danger">without selected bot</span>
-                {% endif %}
+        <div>
+            <div>
+                <ul class="nav nav-tabs md-tabs justify-content-center" id="tabs" role="tablist">
+                    <li class="nav-item">
+                        <a class="nav-link primary-tab-selector {{ '' if use_cloud else 'active show' }}" id="your-profiles-tab" data-toggle="tab" href="#your-profiles" role="tab"
+                           aria-controls="your-profiles"
+                           aria-selected="{{ 'false' if use_cloud else 'true' }}">
+                            <h5>Base & custom</h5>
```

Comment:
What about "built-in" or "Default" or "standard" ?

Reply:
i like "Default"